client/blocks/payment-request/index.js

```diff
@@ -39,6 +39,12 @@ const paymentRequestPaymentMethod = {
 			return false;
 		}
 
+		const shouldShowPaymentRequestButton =
+			getStripeServerData()?.shouldShowPaymentRequestButton ?? false;
+		if ( ! shouldShowPaymentRequestButton ) {
+			return false;
+		}
+
```

Comment:
Could we simplify this a little bit?

```JavaScript
if ( ! getStripeServerData()?.shouldShowPaymentRequestButton ) {
	return false;
}
```

Reply:
Nice catch! Fixed in 25682416b6deac74ba534fc2841082c9cc975af9

I had a bit of trouble trying to make this read a bit better, but you're right; your suggestion is much better 😅